README.md

```diff
@@ -1 +1,132 @@
+
+8.  Go to the “index.html” file in your repository and edit the Title (line 203) and descriptions (line 206) to suit your context. Describe your change and create a new branch for this commit and start a pull request.
+
+    a.  Note: All other descriptions (for lenses, pop up box copy, etc.) are determined by your “standards.csv” file. To make further changes to the description, you must make changes in your Google Sheet and upload a new “standards.csv” file with your changes. We discourage editing in the standards.csv file directly.
+
+9.  Review your changes to “standards.csv” and “index.html” and merge your pull requests with the master branch. (**[How do I merge a pull request?](https://help.github.com/articles/merging-a-pull-request/))**
+
+![Screenshot of merging a pull request in GitHub](media/images/Merge_Pull_Request_GitHub.png)
+
+10.  Share the link for your representation!
+
+![Screenshot of link location in GitHub](media/images/Share_Link_GitHub.png)
+
+**Creating Your Own Digital Literacy Competency Calculator Repository on GitHub**
+------------------------------------------------------------------
+
+GitHub’s documentation is available [here](https://help.github.com/).
+
+1.  Navigate to the repository you want to create.
+
+    a.  ACRL’s Framework:[https://github.com/ccnmtl/diglit-acrl-badge](https://github.com/ccnmtl/diglit-acrl-badge)
+
+    b.  MIT’s Digital Humanities Framework:[https://github.com/ccnmtl/diglit-dh-badge](https://github.com/ccnmtl/diglit-dh-badge)
+
+    c.  Eshet’s Cognitive Skills Framework:[https://github.com/ccnmtl/diglit-cognitive-badge](https://github.com/ccnmtl/diglit-cognitive-badge)
+
+2.  Fork the repository ([**How do I fork a repository?**](https://help.github.com/articles/fork-a-repo/))
+
+3.  Navigate to your repository “Settings”.
+
+    a.  Change the name of the repository. ([**How do I rename a repository?**](https://help.github.com/articles/renaming-a-repository/))
+
+    b.  Scroll down to GitHub Pages. Under Source, select “master branch”. Click Save. Your site with your Digital Literacy Competency Calculator is now published
+
+![Screenshot of publishing a site on GitHub](media/images/Publish_Site_GitHub.png)
+
+4.  Copy the link to your site into the description of your repository for easy reference.
+
+![Screenshot of link location in GitHub](media/images/Share_Link_GitHub.png)
+
+5.  Edit the repository content.
+
+    a.  Tip 1: The recommended workflow is to create a branch from the `master` branch and make edits in that branch. Once you have made your edits, create a pull request. A peer can then review changes and merge your changes with the Master.
+
+    b.  Tip 2: If you are working alone, it is still recommended that you create a pull request with your changes and then merge the changes yourself. Committing directly is NOT recommended because pull requests create a trail of edits that can be followed.
```

Comment:
You can click on the "Files changed" tab at the top, and then click on the 'edit' pencil icon in the top right, and it will prompt to you commit to the `jebrodsky-patch-1` branch, which you can confirm.